src/Organic/Organic.php

```diff
@@ -811,11 +812,13 @@ public function syncPost( $post ) {
             return null;
         }
 
-        $canonical = get_permalink( $post->ID );
+        $canonical = get_permalink( $post );
+        $edit_url = get_edit_post_link( $post );
 
         # In order to support non-standard post metadata, we have a filter for each attribute
         $external_id = \apply_filters( 'organic_post_id', $post->ID );
         $canonical = \apply_filters( 'organic_post_url', $canonical, $post->ID );
+        $featured_image_url = \apply_filters( 'organic_post_featured_image_url', get_the_post_thumbnail_url( $post ), $post->ID );
```

Comment:
New filters should be added to the README
https://github.com/orgnc/wordpress-plugin#filters